doc/getstarted.txt

```diff
@@ -20,14 +14,30 @@ As root, or use sudo.
   % cp doc/uhub.conf /etc/uhub
   % cp doc/users.conf /etc/uhub
```

Comment:
Remove this line, users.conf is deprecated

Reply:
I'll wait @janvidar feedback / correction